content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,296 @@
+`_type`                 | No        | [`string`]  | A string of comma-delimited FHIR resource types. Example: `_type=Patient,Encounter,Location`
+`_typeFilter`           | No        | [`string`]  | A string of comma-delimited FHIR Rest queries provided in combination with `_type`. Example: `_typeFilter=Condition?clinical-status=active,Encounter?status=planned`
+`_since`                | No        | [`instant`] | Resources updated after this time will be included in the response. Example: `2021-03-18T19:19:42.000Z`
+`includeAssociatedData` | No        | [`string`]  | When provided, the group export will return or omit a pre-defined set of FHIR resources associated with the request. A string of comma-delimited values. Example: `includeAssociatedData=LatestProvenanceResources`
+`_outputFormat`         | No        | [`string`]  | The format for the requested bulk data files to be generated. Example: `_outputFormat=application/ndjson`
+
+_Notes_
+
+* `_type` valid values are `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `Device`, `DiagnosticReport`, `DocumentReference`, `Encounter`, `Goal`, `Immunization`, `Location`, `MedicationRequest`, `Patient`, `Procedure`, `Observation`, `Organization`, `Practitioner`, `Provenance`
+* When the `_type` parameter contains a reference resource (`Location`, `Organization`, `Practitioner`) or associated resource (`Provenance`), at least one of the following resources must also be provided: `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `Device`, `DiagnosticReport`, `DocumentReference`, `Encounter`, `Goal`, `Immunization`, `MedicationRequest`, `Observation`, `Patient`, `Procedure`
+* When the `_type` parameter is not provided, the resources to be exported will be determined by the scopes provided in the authorization token.
+* When the `_typeFilter` parameter is provided, the `_type` parameter MUST be provided as well and must only have queries for the resources provided in `_type`
+* `_typeFilter` does not support queries for the following resources: `Location`, `Organization`, `Patient`, `Practitioner`, `Provenance`
+* `_typeFilter` does not support queries with the following search parameters: `_count`, `_contained`, `_containedType`, `_elements`, `_id`, `_include`, `_lastUpdated`, `_revinclude`, `_sort`, `_summary`, `_total`, `encounter`, `identifier`, `patient`, `subject`
+* `_typeFilter` supports queries with the search parameters supported for the [R4](/millennium/r4) resources.
+* Supplying `_typeFilter` with the same resource multiple times is not supported.
+* `includeAssociatedData` valid parameters are `LatestProvenanceResources` and `RelevantProvenanceResources`
+* `_outputFormat` valid parameters are `application/fhir+ndjson`, `application/ndjson`, `ndjson`
+* `_outputFormat` defaults to `application/fhir+ndjson` when not provided.
+* The following parameters are not supported: `_elements`, `patient`
```

Comment:
realized do we need this as those are not params in the V1 spec?

Reply:
The note that we support experimental parameters from V2 has been added here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R39.

Updated to say that the following experimental parameters are not supported: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R39